tests/campaigns/test_integration.py

```diff
@@ -34,11 +39,45 @@ def assert_sub_returned(self, response, campaign_id, budget,
         self.assertEqual(response['subscription']['budget'], budget)
 
 
+class SearchMixin(object):
+    campaign_type = 'search'
+
+    @classmethod
+    def subscribe_method(cls):
```

Comment:
We could probably make this as class level attr (the same as with `campaign_type`).  Just a bit more concise.